fenix/app/src/main/res/values-night/colors.xml

```diff
@@ -151,4 +151,7 @@
     <!-- Tab Counter colors -->
     <color name="mozac_ui_tabcounter_default_tint" tools:ignore="UnusedResources">@color/fx_mobile_icon_color_primary</color>
     <color name="mozac_ui_tabcounter_default_text" tools:ignore="UnusedResources">@color/fx_mobile_text_color_primary</color>
+
+    <!-- Cookie Banner Handling -->
+    <color name="cookie_banner_handling_details_report_a_site_text_color">@color/fx_mobile_text_color_accent</color>
```

Comment:
Instead of adding this new colour, lets use directly in the UI `fx_mobile_text_color_accent , see https://github.com/mozilla-mobile/firefox-android/pull/1298#discussion_r1152540254 for more details. Unless there is a specific use case for adding the new colour, that I maybe missing, in that case help to clarify :)  ?

Reply:
Done, I removed the color.